src/static/css/2019.css

```
@@ -1348,6 +1361,45 @@ p.copyright a {
   min-width: 100%;
 }
 
+.nav-dropdown-list-part {
+  white-space: nowrap;
+  padding: 16px 8px 8px 8px;
```

Add some extra padding on the left:

```suggestion
  padding: 16px 8px 8px 16px;
```